server/bleep/src/ctags.rs

```diff
@@ -118,7 +118,10 @@ async fn call_ctags(paths: Vec<String>, exclude_langs_list: &[&str]) -> Result<S
     Ok(String::from_utf8(output.stdout)?)
 }
 
-pub async fn get_symbols(repo_disk_path: &Path, exclude_langs: &[&str]) -> SymbolMap {
+pub async fn get_symbols_with_ctags(
```

Comment:
this suffix feels redundant since it already belongs to the `ctags` module, and is often invoked with a qualified path, such as `ctags::get_symbols` (as done in `benches/indexes.rs`).

Reply:
Yes, that's fair enough. Will make sure that they're always invoked with their qualified paths.